### PR TITLE
Bugfix/562 Runtime error when Auth Library validates manually generated ID tokens

### PR DESCRIPTION
**Pull request by ywkim312**
Sandeep;
Could you test it with the event thing that you had an error. I tested it in my local using auth-test but I did not know how to replicate the error in event but this should work.

### CODE REVIEW

**Comment by sandeep-ps** [comment]
@ywkim312, please update the CHANGELOG. Thanks.

**Comment by ywkim312** [comment]
> @ywkim312, please update the CHANGELOG. Thanks.

Sorry. Updated

File: lib/auth-middleware/auth_middleware/__init__.py [hunk 1]
```diff
@@ -210,7 +210,8 @@ def verify_userauth(id_token, group_name=None, internal_token_only=False):
             #            file1.close()
             lines = base64.b64decode(os.getenv('ROKWIRE_PUB_KEY'))
             keyset = json.loads(lines)
-            target_client_id = os.getenv('ROKWIRE_API_CLIENT_ID')
+
```
Comment (sandeep-ps) [inline lib/auth-middleware/auth_middleware/__init__.py:213]: Line number 199 needs to be updated to `target_client_ids=None`

Comment (ywkim312) [reply lib/auth-middleware/auth_middleware/__init__.py:213]: Sorry, I missed it. Updated

Comment (sandeep-ps) [reply lib/auth-middleware/auth_middleware/__init__.py:213]: No issues. Thanks.

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**